SPECS/expat.spec

```diff
@@ -60,334 +62,30 @@ make check
 %doc AUTHORS Changes
 %license COPYING
 %{_bindir}/*
-%{_libdir}/lib*.so.*
-%{_mandir}/*/*
+%{_libdir}/libexpat.so.1
+%{_libdir}/libexpat.so.1.*
+%if 0%{?xenserver} < 9
+%{_docdir}/%{name}/AUTHORS
+%{_docdir}/%{name}/Changes
```

Comment:
I really don't see the point. They already have a `%doc` on these files. Do you see a difference in the package listing when comparing with or without this? Maybe what it changes is that the files are not referenced as doc anymore at the RPM metadata level and they want this for some reason?

Reply:
`%doc` installs the files in `/usr/share/doc/expat-2.5.0/`, while those lines are packaging the files that are installed by `make install` in `/usr/share/doc/expat/`.
I'm not sure why they need both, but that doesn't seem like a problem either.